data/enemy/base.py

```diff
@@ -0,0 +1,16 @@
+from typing import Self
+
+
+class BaseEnemy:
+    def __init__(self: Self) -> Self:
+        super().__init__()
+        self.name = "UnknownEnemy"
+        self.guid = "NONE"
+        self.hp = 0
+        self.speed = 0
+        self.physical_defense = 0
+        self.physical_attack = 0
+        self.magic_defense = 0
+        self.magic_attack = 0
+        self.level = 0
+        self.fleshmancer_minion = False
```

Comment:
My one gripe with the data is that every single enemy appears to be set to `fleshmancer_minion = True`, which doesn't seem correct. Unless there's a grand conspiracy or something, and Wyrd is actually an agent of the Fleshmancer O_o

Reply:
probably a mistake!